scripts/generate-certificates.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
please turn all of the bash scripts into golang fixtures for the tests

Reply:
done